hawc/apps/assessment/migrations/0030_assessment_epi_version.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.10 on 2022-01-03 20:11
```

Comment:
let's just create a single migration for this.  The default should be 2, but you may need to add a [RunPython](https://docs.djangoproject.com/en/4.0/ref/migration-operations/#django.db.migrations.operations.RunPython) to set the existing values to 1.